desktop/src/main/java/bisq/desktop/util/validation/SecurityDepositValidator.java

```diff
@@ -17,21 +17,28 @@
 
 package bisq.desktop.util.validation;
 
-import bisq.core.btc.wallet.Restrictions;
 import bisq.core.locale.Res;
+import bisq.core.payment.PaymentAccount;
 import bisq.core.util.BSFormatter;
 
 import javax.inject.Inject;
 
+import static bisq.core.btc.wallet.Restrictions.getMaxBuyerSecurityDepositAsPercent;
+import static bisq.core.btc.wallet.Restrictions.getMinBuyerSecurityDepositAsPercent;
```

Comment:
More static imports

Reply:
Here it doesn't help to make the code more readable. Commit below.